easybuild/easyconfigs/m/Mako/Mako-1.0.7-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae']
+
+download_dep_fail = True
+use_pip = True
+
+builddependencies = [('binutils', '2.31.1')]
+dependencies = [('Python', '3.7.2')]
```

Comment:
@Micket Any reason you're not using `multi_deps` here?

`mako-render` is just a Python script, so there's no concern w.r.t. a multi-Python installation...